eve/main.yml

```diff
@@ -458,6 +459,7 @@ stages:
           stage_names:
             - single-node-install-rhel
             - multiple-nodes-centos
+            - multiple-nodes-example-solution-centos #remove me
```

Comment:
Do we really need a new environment to test solution ?? ( For me no) Couldn't be part of the multi node we already have ?

Reply:
No, because we will do it in postmerge.

Reply:
Hmm yes true, then you also need to add like for all other step the right env variable (and add these env variable on `add_final_stauts` like for all other steps)
```
_metalk8s_internal_info:
      junit_info: &_install_multi-node_solution_junit_info
....
```
To generate the right junit result file.

Reply:
I don't think it should be a "multi nodes" cluster. IMO, we want:

- one Bootstrap, to install Metal, and then import + run the Solution 
- one extra VM to run Cypress tests

This should also be the case for "single node" tests, so yeah, maybe merge them.

Reply:
Then you need to remove taint on the bootstrap node, but it's doable yes (or add toleration for example solution)

Reply:
Adding taints on the Bootstrap node was handle here: https://github.com/scality/metalk8s/pull/2321/files#diff-73e74177595a435f88a0701e605cf08eR1686-R1698

Reply:
Hmmm then why you need multi node ?
Anyway I agree with @gdemonet comment https://github.com/scality/metalk8s/pull/2321#pullrequestreview-383610719 this test should be directly in pytest